### PR TITLE
Add simple prowlarr

### PR DESCRIPTION
**Pull request by halkeye**
Super basic prowlarr support - #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +125,19 @@ Visit http://127.0.0.1:9708/metrics to see Radarr metrics
...
+```sh
+./exportarr prowlarr --help
+
+./exportarr prowlarr \
+  --port 9710 \
+  --url http://x.x.x.x:9696 \
+  --api-key amlmndfb503rfqaa5ln5hj5qkmu3hy18
+```
+
+Visit http://127.0.0.1:9709/metrics to see Radarr metrics
```
Comment (onedr0p) [inline README.md:139]:
```suggestion
Visit http://127.0.0.1:9709/metrics to see Prowlarr metrics
```

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ See examples in the [examples/compose](./examples/compose/) directory
 See examples in the [examples/kubernetes](./examples/kubernetes/)
 directory.
 
-This exporter is also included in the Lidarr, Radarr, and Sonarr helm
+This exporter is also included in the Lidarr, Radarr, Sonarr, and Prolarr helm
```
Comment (onedr0p) [inline README.md:26]:
```suggestion
This exporter is also included in the Lidarr, Radarr, Sonarr, and Prowlarr helm
```

**Review by onedr0p — COMMENTED** [review]
Looks good so far @halkeye

**Comment by halkeye** [comment]
This meets my needs for now. I mostly want alerts on health check issues

**Comment by onedr0p** [comment]
Cool, I'll pull this down and try to run it. If there's no issues I'll merge it and create a release.

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
@halkeye I ran this locally, I wonder if there's a better way to format the failures? `prowlarr_system_health_issues` will be a very large metric to parse as labels.

```
prowlarr_history_total{url="http://prowlarr"} 1.004175e+06
prowlarr_system_health_issues{message="Indexers unavailable due to failures for more than 6 hours: SecretSite1, SecretSite2, SecretSite3",source="IndexerLongTermStatusCheck",type="warning",url="http://prowlarr",wikiurl="https://wiki.servarr.com/prowlarr/system#indexers-are-unavailable-due-to-failures"} 1
prowlarr_system_health_issues{message="Indexers unavailable due to failures: Redacted",source="IndexerStatusCheck",type="warning",url="http://prowlarr",wikiurl="https://wiki.servarr.com/prowlarr/system#indexers-are-unavailable-due-to-failures"} 1
prowlarr_system_status{url="http://prowlarr"} 1
```

Should we export all the indexers and have a status? e.g. where 0 is success and 1 is failure?

```
prowlarr_indexer_status{indexer="SecretSite1"} 1
prowlarr_indexer_status{indexer="SecretSite2"} 0
```

**Comment by halkeye** [comment]
I honestly don't know. I'm super new to prometheus and I have prowlarr deployed somewhere I have to manually update, so i mostly just want alerts when its out of date. Anything else is just a bonus

**Comment by onedr0p** [comment]
I can take a look at it when I have time, no worries! But if you don't hear about it for awhile and have time give it a shot ;)

**Comment by halkeye** [comment]
@onedr0p any chance you'll do a release any time soon?

**Comment by onedr0p** [comment]
It would be nice to have more metrics collected from prowlarr before I make a release, you can continue to use the main/master container tag.

**Pull request merged**